SupportClient/SupportClient/Extensions/UIView+Extensions.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UIView+Extensions.swift
+//  SupportClient
+//
+//  Created by Cameron Russell on 3/6/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+
+// Resolves DZNEmptyDataSet issue #263
```

Comment:
I see a little better now. We should still create other view classes though for our empty state views so we don't clutter up our view controller files